### PR TITLE
Mention including cluster state archive in GitHub issues

### PR DESCRIPTION
**Pull request by Tahler**

### CODE REVIEW

File: content/help/bugs/index.md [hunk 1]
```diff
@@ -9,8 +9,48 @@ aliases:
...
-Search our [issue database](https://github.com/istio/istio/issues/) to see if we already know about
-your problem and learn about when
-we think we can fix it. If you don't find your problem in the database, please open a [new issue](https://github.com/istio/istio/issues/new/choose)
-and let us know what's
-going on.
+Search our [issue database](https://github.com/istio/istio/issues/) to see if
+we already know about your problem and learn about when we think we can fix
+it. If you don't find your problem in the database, please open a [new
+issue](https://github.com/istio/istio/issues/new/choose) and let us know
+what's going on. Be sure to include a [cluster state
```
Comment (geeknoid) [inline content/help/bugs/index.md]: I suggest creating a separate paragraph for the last sentence.

I suggest starting this paragraph with:

If you're running on Kubernetes, consider including a...

File: content/help/bugs/index.md [hunk 2]
```diff
@@ -9,8 +9,48 @@ aliases:
...
+
+* Run via `curl`:
+
+  ```sh
+  curl https://raw.githubusercontent.com/istio/istio/release-1.0/tools/dump_kubernetes.sh | sh -s -- -z
+  ```
+
+* Run locally, from the release directory's root:
+
+  ```sh
```
Comment (geeknoid) [inline content/help/bugs/index.md]: Use {{< text bash >}} here, and indent by 4 chars.

See https://preliminary.istio.io/about/contribute/writing-a-new-topic/#embedding-preformatted-blocks for info.

Comment (geeknoid) [reply content/help/bugs/index.md]: Can also use the @@ model to generate a link to the .sh file on GitHub. See the same doc page for info on that feature.

File: content/help/bugs/index.md [hunk 3]
```diff
@@ -9,8 +9,50 @@ aliases:
...
+
+## Generating a cluster state archive
+
+For convenience, you can run a dump script to produce an archive containing
+all of the needed state from your Kubernetes cluster:
+
+* Run via `curl`:
+
+    {{< text bash >}}
+    $ curl https://raw.githubusercontent.com/istio/istio/release-1.0/tools/dump_kubernetes.sh | sh -s -- -z
```
Comment (geeknoid) [inline content/help/bugs/index.md]: replace release-1.0 with {{< branch_name >}}. This will insert the appropriate branch name for the current release.

Comment (Tahler) [reply content/help/bugs/index.md]: Oh nice. Done.

**Pull request merged**